llvm/lib/Transforms/Vectorize/VPlanRecipes.cpp

```diff
@@ -3367,6 +3427,8 @@ void VPFirstOrderRecurrencePHIRecipe::print(raw_ostream &O, const Twine &Indent,
 void VPReductionPHIRecipe::execute(VPTransformState &State) {
   auto &Builder = State.Builder;
 
+  auto VF = State.VF.divideCoefficientBy(VFScaleFactor);
```

Comment:
This (and a new test) are the only changes to the landed diff, right? Could you add a comment about the scaling here?

Maybe spell out VFs type to make it obvious if it the whole line stays < 80 chars
```suggestion
  ElementCount VF = State.VF.divideCoefficientBy(VFScaleFactor);
```

Reply:
This line wasn't changed, but the second commit is the only difference to the landed diff. The first commit and the landed one would have the same commit ID but I squashed all of the fixup commits from the old branch first.

A comment is a great idea, done!